docs/sdk/client_user_guide.rst

```diff
@@ -292,114 +283,19 @@ via the ``authorizer`` parameter:
       'https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/all'
 
 
-.. _specifying-serde-strategy:
-
-Specifying a Submit-Side Serialization Strategy
------------------------------------------------
-
-When sending functions and arguments for execution on a Compute endpoint, the SDK uses
-the :class:`~globus_compute_sdk.serialize.ComputeSerializer` class to convert data to and from a format that can be easily
-sent over the wire. Internally, :class:`~globus_compute_sdk.serialize.ComputeSerializer` uses instances of
-:class:`~globus_compute_sdk.serialize.SerializationStrategy` to do the actual work of serializing (converting function code
-and arguments to strings) and deserializing (converting well-formatted strings back into
-function code and arguments).
-
-The default strategies are :class:`~globus_compute_sdk.serialize.DillCode` for function code and :class:`~globus_compute_sdk.serialize.DillDataBase64` for
-function ``*args`` and ``**kwargs``, which are both wrappers around |dill|_. To choose
-another serializer, use the ``serializer`` member of the Compute :class:`~globus_compute_sdk.Executor`:
-
-.. code:: python
-
-  from globus_compute_sdk import Executor
-  from globus_compute_sdk.serialize import ComputeSerializer, CombinedCode, JSONData
-
-  with Executor('<your-endpoint-id>') as gcx:
-    gcx.serializer = ComputeSerializer(
-      strategy_code=CombinedCode(), strategy_data=JSONData()
-    )
-
-:doc:`See here for a up-to-date list of serialization strategies. </reference/serialization_strategies>`
+Endpoint operations
+-------------------
 
-To check whether a strategy works for a given use-case, use the :func:`~globus_compute_sdk.serialize.ComputeSerializer.check_strategies`
-method:
+You can retrieve information about endpoints including status and
+information about how the endpoint is configured.
```

Comment:
The example shows the status, but not "how the endpoint is configured."  Had you intended to also show `get_endpoint_metadata()`?